content/rma_intro.tex

```diff
@@ -16,31 +16,65 @@
 one-sided communication interfaces via \Celev{} generic selection 
 (\Celev{} \S6.5.1.1\footnote{Formally, the \Celev{} specification is ISO/IEC 9899:2011(E).})
 for block, scalar, and block-strided put and get communication. 
-Such type-generic routines are supported for the ``standard \ac{RMA} types’’
-identified in Table \ref{stdrmatypes}.
+Such type-generic routines are supported for the
+\newtext{\Celev{}
+``standard integer types'' (\Celev{} \S6.2.5.7),
+``real floating types'' (\Celev{} \S6.2.5.10),
+``exact-width integer types'' (\Celev{} \S7.20.1.1),
+``integer types capable of holding object pointers'' (\Celev{} \S7.20.1.4),
+and the integer types \textit{size\_t} and \textit{ptrdiff\_t} defined in
+stddef.h (\Celev{} \S7.19), which are collectively called the}
+``standard \ac{RMA} types''
+\newtext{and are listed} \oldtext{identified} in Table \ref{stdrmatypes}.
+\newtext{Implementations may optionally support additional types.}
+
+\textbf{\newtext{Note to implementors:}}
+\begin{itemize}
+\item \newtext{%
+  In order to provide compatibility between C and C++, implementors may elect
+  to include a dependence on stdbool.h, which provides the macro \textit{bool}.
+  Alternatively, implementors may elect to avoid this dependence and use the
+  C99\footnote{Formally, the C99 specification is ISO/IEC 9899:1999(E), including amendments and corrections.}
+  \textit{\_Bool} type explicitly.
+  }
```

Comment:
I think what I'm trying to ask for here is that we define what happens when you pass ```shmem.h``` through a C89 compiler.  We now have multiple types used in the spec, but not defined prior to C99.  So, what should happen?  Am I allowed to disable these bindings in C89 mode so that this will work?  Or, should we require C99 as a baseline?